finance/clearifying_question_to_moore.txt

```diff
@@ -0,0 +1,32 @@
+Dear [INSERT CONTACT AT MOORE FOUNDATION],
+
+thank you so much for giving the grant "Sustaining and Growing the Astropy
+Project" to us. As you know from the grant application, we are still working
+out a formal government and management structure, because the astropy project
+has not received grants like this before.
+
+However, many astropy members are used to government issued contracts and
+grants (e.g. NSF, NASA) from their non-astropy science work, where every grant
+comes with a separate set of rules.
+
+We'd thus like to clarify a few points about timeline and spending about the
+Moore grant early on - in particular since you awarded us more money than we
+originally budgeted and we are trying to decide now, how that extra money will
+benefit the project most.
+
+- We proposed work over three years of time. When did the clock for that
+  formally start?  
+
+- We are keen to get the work done, but if we don't spend
```

Comment:
Should probably clarify that this is a no-cost extension.

Reply:
I was trying to avoid wording that has a specific legal meaning in NASA/NSF context.